code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -281,6 +281,9 @@
 			if(DISGUST_LEVEL_DISGUSTED to INFINITY)
 				msg += "[t_He] look[p_s()] extremely disgusted.\n"
 
+	var/datum/quirk/allergic/allergen = has_quirk(/datum/quirk/allergic)
+	if((allergen && reagents?.has_reagent(allergen.reagent_id)) || reagents?.has_reagent(/datum/reagent/toxin/histamine))
```

Comment:
could also reference the cooldown that the quirk uses to track allergic reactions

Reply:
that would be strange and inconsistent use of a variable that is not meant for that